### DISCUSSION TITLE
no solution loaded sometime except only a gray bracket

### DISCUSSION THREAD

**Discussion by yifanff** [community/community#10720] [Copilot Conversations]
Why sometimes no solution loaded except only a gray bracket?

![image](https://user-images.githubusercontent.com/5958693/151653249-a90872fe-51e7-4c15-8b08-6dec8b524968.png)

**Comment by davecheney** [comment]
@yifanff  Try renaming `end` to `e`

**Reply by yifanff** [reply]
Thanks for quick feedback, rename end to e, not work.
![image](https://user-images.githubusercontent.com/5958693/151655853-c0fe5266-f6c0-43fb-ae65-7acbbc94c27d.png)

**Reply by davecheney** [reply]
ok, thanks for confirming
